composer.json

```diff
@@ -22,24 +22,27 @@
     "require": {
         "php": ">=5.5.0",
         "payum/core": "^1.3",
-        "omnipay/common": "^2.3"
+        "omnipay/common": "^3.0",
```

Comment:
^3.0@alpha since there is not stable version for now.

Reply:
right, changed